### PR TITLE
Update share-to-teams.md

### PR DESCRIPTION
**Pull request by pallabiwrites**
Fixed minor language issues.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/share-to-teams.md](https://github.com/MicrosoftDocs/msteams-docs/blob/63b017c93b739cbd492ecbfaf06a3d04de668b3d/msteams-platform/concepts/build-and-test/share-to-teams.md) | 87 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a670133a-fbbe-4702-b57c-f32584b9ac88) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/build-and-test/share-to-teams.md [hunk 1]
```diff
@@ -57,11 +57,11 @@ The following image displays the Share-to-Teams pop-up experience:
     </div>
     ```
 
-1. If your page dynamically renders content, you can use the the `shareToMicrosoftTeams.renderButtons()` method to force the **Share** button to render at the appropriate place in the pipeline.
+1. If your page dynamically renders content, you can use the `shareToMicrosoftTeams.renderButtons()` method to force the **Share** button to render at the appropriate place in the pipeline.
 
 ## Craft your website preview
 
-When your website is shared to Teams, the card that is inserted into the selected channel contains a preview of your website. You can control the behavior of this preview by ensuring the appropriate meta-data is added to the website being shared, such as the `data-href` URL.  
+When your website is shared to Teams, the card that is inserted into the selected channel contains a preview of your website. You can control the behavior of this preview by ensuring the appropriate metadata is added to the website being shared, such as the `data-href` URL.
```
Comment (v-abirade) [inline msteams-platform/concepts/build-and-test/share-to-teams.md:64]: As per MSTP, meta is hyphenated, please change to meta-data.

File: msteams-platform/concepts/build-and-test/share-to-teams.md [hunk 2]
```diff
@@ -57,11 +57,11 @@ The following image displays the Share-to-Teams pop-up experience:
     </div>
     ```
 
-1. If your page dynamically renders content, you can use the the `shareToMicrosoftTeams.renderButtons()` method to force the **Share** button to render at the appropriate place in the pipeline.
+1. If your page dynamically renders content, you can use the `shareToMicrosoftTeams.renderButtons()` method to force the **Share** button to render at the appropriate place in the pipeline.
```
Comment (v-abirade) [inline msteams-platform/concepts/build-and-test/share-to-teams.md:60]: If your page dynamically renders content, you can use the `shareToMicrosoftTeams.renderButtons()` method to force **Share** to render at the appropriate place in the pipeline.

**Review by v-abirade — CHANGES_REQUESTED** [review]
Please make the changes.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/share-to-teams.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ffa3909fc7923f1e3ea0b56e87243e960a7941d3/msteams-platform/concepts/build-and-test/share-to-teams.md) | 87 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b0c6a7d6-7782-4df4-9555-2c815fb25425) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/build-and-test/share-to-teams.md (hunk 2 above)
Comment (pallabiwrites) [reply msteams-platform/concepts/build-and-test/share-to-teams.md]: Fixed.

File: msteams-platform/concepts/build-and-test/share-to-teams.md (hunk 1 above)
Comment (pallabiwrites) [reply msteams-platform/concepts/build-and-test/share-to-teams.md]: Fixed.

**Review by v-abirade — APPROVED** [review]
Approved

**Pull request merged**